csync/src/csync_exclude.c

```diff
@@ -102,7 +107,7 @@ int csync_exclude_load(const char *fname, c_strlist_t **list) {
   int fd = -1;
   int i = 0;
   int rc = -1;
-  int64_t size;
+  off_t size;
```

Comment:
We have had problems with off_t before  (see commit 4f5bfbdb784fb4c589ff04c439fb82749b3b54e5 )